cypress/integration/login.js

```diff
@@ -1,28 +1,114 @@
+import LoginPage from '../objects/logingPage'
+import TimeLogingPage from '../objects/timeLogingPage'
+const loginPage = new LoginPage()
+const timeLogingPage = new TimeLogingPage()
 describe('Sourcebooks login', function() {

```

Comment:
Feels like this place could use an empty new line to separate these variables from the describe block